src/pgduckdb_background_worker.cpp

```diff
@@ -417,11 +451,9 @@ TriggerActivity(void) {
 	}
 
 	SpinLockAcquire(&BgwShmemStruct->lock);
-	BgwShmemStruct->activity_count++;
-	/* Force wake up the background worker */
-	if (BgwShmemStruct->bgw_latch) {
```

Comment:
This `if` meant that somehow we were notifying changes before the bgw was available.
This should not happen anymore.